andela/ah-backend-invictus
# #165273479 Add Comment CRUD

### **What does this PR do?**
This PR adds the user ability to;
- Create comment
- Read comments
- Read a comment
- Update comment
- Delete comment
### **How to test it?**
Make migrations, run the server and try out the following endpoints in postman.

- `http://127.0.0.1:8000/api/articles/<article_id>/comments/` to create and read comments.
- `http://127.0.0.1:8000/api/articles/<article_id>/comments/<comment_id>` to read, update and delete a comment.

Example Request body;

```
{
    "comment": {
        "body": "His name was my name too."
    }
}
```
Expected JSON response body;
```
{
    "comment": {
        "id": 11,
        "author": "mwinel2018",
        "body": "His name was my name too.",
        "created_at": "2019-05-01T13:56:47.542996Z",
        "updated_at": "2019-05-01T13:56:47.543017Z",
        "article": 1
    },
    "message": "Comment successfully created."
}
```

### **What are the relevant user stories**
[#165273479](https://www.pivotaltracker.com/story/show/165273479)
